docs/src/references/certificate-management.md

```diff
@@ -0,0 +1,358 @@
+                                         Example: 365d
+  certificate.validity.minimum_duration  Minimum validity duration below which a new certificate should be requested.
+                                         Note: This is an advisory setting and the renewal has to be scheduled
+                                         Example: 30d
+               certificate.organization  Organization name used for certificate signing requests.
+                                         Example: ACME
+          certificate.organization_unit  Organization unit used for certificate signing requests.
+                                         Example: IoT
+```
+
+### certificate.validity.minimum_duration
+
+The minimum duration setting controls when the exit-code of the `tedge cert needs-renewal` command allowing users to programmatically check if the certificate is "about to expire" and hence can trigger request a new certificate.
+
+:::caution
+If it is normal for devices to be disconnected from the cloud for extended periods of time (e.g. > 1 week), then it is strongly recommend that the minimum duration (`certificate.validity.minimum_duration`) be at least 3 times the maximum offline period to ensure that the device has enough time to renew its certificate whilst the device is able to communicate with Cumulocity. Otherwise, the device is at risk of failing to renew its certificate before it expires which would require the device to go through the [registration process](#device-registration) again.
+:::
+
+## Certificate renewal
+
+Once the %%te%% device is connected to Cumulocity, its certificate can be renewed by using the existing connection to Cumulocity. If the device is using SystemD as its init system, then the certificate certificate renewal process will be automatic, and it will be renewed before the certificate expires (the default is 30 days before expiration, but the value can be configured via the tedge configuration).
+
+The Cumulocity certificate-authority feature provides a new API endpoint for devices to request a new certificate provided the device can still communicate with the tenant it is requesting a certificate from. The certificate renewal is authenticated by way of a JWT which is requested over the MQTT connection using the current certificate. If for any reason the certificate expires, and the device can no longer communicate with Cumulocity, then the device will need to be registered again.
+
+To ensure a smooth transition to the new certificate, the certificate renewal process is done in two distinct steps as described below:
+
+1. Request a new certificate authenticating with the current certificate. The newly issued certificate is downloaded to the device, however it will not overwrite the current certificate. Instead the newly issued certificate will be saved next next to the current certificate and be given the `.new` filename suffix.
```

Comment:
```suggestion
1. Request a new certificate authenticating with the current certificate. The newly issued certificate is downloaded to the device, however it will not overwrite the current certificate. Instead the newly issued certificate will be saved next to the current certificate and be given the `.new` filename suffix.
```

Reply:
Fixed in f3a099ff38ed6949b893fdb6e704fa013c82df16